doc/getting_started/installation_win.rst

```diff
@@ -87,6 +87,35 @@ packages from their respective websites.
       cd %userprofile%\zephyr
       pip3 install --user -r scripts/requirements.txt
 
+#. If you want to be able to run the ``menuconfig`` interactive configuration
+   interface, first run the following command to check which Python version you
+   are using, and whether it's 32- or 64-bit. (**Note: Do not assume that you
+   are running a 64-bit version of Python even if your OS is 64-bit.**)
+
+   .. code-block:: console
+
+      C:\...\zephyr> python -VV
```

Comment:
remove the prompt `c:\...\zephyr>`

Reply:
IMO, that makes it less clear that what follows it is the output, but alright.

Reply:
yes, but makes it harder to copy-paste and we have this convention all over the documentation

Reply:
Hmm... yeah, think I would want to split it up in that case.

I think I prefer for it to be immediately obvious rather than perfectly consistent. What do you think?

Reply:
Then do it as follows:

Type the following command:
```
python -VV
```
and check for the output, which should be something like:;
```
Python 3.6.5 (v3.6.5:f59c0932b4, Mar 28 2018, 16:07:46) [MSC v.1900 32 bit (Intel)]
```